packages/protobuf/src/google/protobuf/any_pb.ts

```diff
@@ -216,6 +217,17 @@ export class Any extends Message<Any> {
     return true;
   }
 
+  unpack(registry: IMessageTypeRegistry): Message | undefined {
+    if (this.typeUrl === "") {
+      return undefined;
+    }
+    const messageType = registry.findMessage(this.typeUrlToName(this.typeUrl));
+    if (!messageType) {
+      throw new Error(`cannot unpack message: ${this.typeUrl} is not in the type registry`);
+    }
```

Comment:
Is it the right call to raise an error if the message wasn't found?

Users will already have to handle the `undefined` return value already for the case that the Any is empty. I have the gut feeling that it's best to only have users handle one error path, and return `undefined` if the message type is not in the registry.

Reply:
I had the same thought actually.  I figured it was keeping consistent with the `is` implementation, which returns false on an empty `Any`, but can throw an error with a bad type URL (see `typeUrlToName`).  Should we also change that or does an error in that instance make more sense?

Reply:
Sounds good.  Updated!